src/OhMyThreads.jl

```diff
@@ -6,7 +6,12 @@ for mac in Symbol.(["@spawn", "@spawnat", "@fetch", "@fetchfrom"])
 end
 
 using ChunkSplitters: ChunkSplitters
+const index_chunks = ChunkSplitters.index_chunks
 const chunks = ChunkSplitters.chunks
+const Split = ChunkSplitters.Split
+const Consecutive = ChunkSplitters.Consecutive
+const RoundRobin = ChunkSplitters.RoundRobin
```

Comment:
I think its better if this was 
```suggestion
using ChunkSplitters: ChunkSplitters, index_chunks, chunks, Split, Consecutive, RoundRobin
```
because then they'll print nicer. Here's a demo of what I mean: 

Compare
```julia
julia> module Outer

       module Inner
       struct Foo end
       end

       const Foo = Inner.Foo
       export Foo
       end;

julia> using .Outer

julia> Foo
Main.Outer.Inner.Foo
```
vs
```julia
julia> module Outer2

       module Inner
       struct Bar end
       end

       using .Inner: Bar
       export Bar
       end;

julia> using .Outer2

julia> Bar
Bar
```

Reply:
Interestingly, I can't reproduce this for our use-case (although I can reproduce your demo). What I find is this:

**via using**

```julia-repl
julia> using OhMyThreads

julia> OhMyThreads.Consecutive() # does NOT auto-complete!
ChunkSplitters.Consecutive()
```

**as const**

```julia-repl
julia> using OhMyThreads

julia> OhMyThreads.Consecutive() # does auto-complete
ChunkSplitters.Consecutive()
```

Irrespective of the printing, it seems that for auto-completion to work (at least for the types) we need them as `const`s. (Note that we also don't export the split types.)

Reply:
Also note that you actually proposed to use the `const` pattern for `TaskLocalValue`, IIRC precisely to make auto-completion work.

Reply:
Oh interesting! Yeah you're right, I had forgotten about this subtle difference. Nevermind then!